### ISSUE TITLE
[Consumer] Update json api request to json api changes

### ISSUE THREAD

**Issue by Lowell20** [codeforeauclaire/foodtrucks#22]
I've cleaned up the json output to only show events with their associated food truck info.
Please adjust the request if needed.

**Comment by AnthonyAstige** [comment]
@Lowell20 What changeset was this done in (so I can see what changed)?

**Comment by AnthonyAstige** [comment]
Assigned to Levi, looks like he may have started this as noted in code comment above.

**Comment by Lowell20** [comment]
here is a sample of the data structure as it is now. I'm not sure if it is any different than before. I created this issue to let you know there may be a difference, just in case. Sorry about the ambiguity here.
![2016-05-23_1828](https://cloud.githubusercontent.com/assets/1144169/15486521/7838690a-2114-11e6-9387-819cab79b540.png)

I made some big changes in how the data is collected and combined, but hopefully not too many changes in how it is rendering here

http://foodtrucks.version-three.com/default-endpoint
